metrics.sh

```diff
@@ -4,6 +4,7 @@
 LC_ALL=en_US.UTF-8
 LANG=en_US.UTF-8
 LANGUAGE=en_US.UTF-8
+DIR=$(dirname "$(readlink -f "$0")")
```

Comment:
Great PR! Please remove the `readlink` part, as this command is not always available. 
I’d go with a simple `PWD=$(dirname "$0")` for our use case.